pkgs/applications/misc/passwordsafe/default.nix

```diff
@@ -0,0 +1,65 @@
+{ stdenv, meson, ninja, pkg-config, gettext, fetchFromGitLab
+, python3, libhandy, libpwquality
+, wrapGAppsHook, gtk3, glib, gobject-introspection
+, desktop-file-utils, appstream-glib, gnome3 }:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "passwordsafe";
+  version = "3.99.2";
+
+  format = "other";
+
+  src = fetchFromGitLab {
+    domain = "gitlab.gnome.org";
+    owner = "World";
+    repo = "PasswordSafe";
+    rev = version;
+    sha256 = "0pi2l4gwf8paxm858mxrcsk5nr0c0zw5ycax40mghndb6b1qmmhf";
+  };
+
+  nativeBuildInputs = [
+    meson
+    ninja
+    gettext
+    pkg-config
+    wrapGAppsHook
+    desktop-file-utils
+    appstream-glib
+    gobject-introspection
+  ];
+
+  buildInputs = [
+    gtk3
+    glib
+    gnome3.adwaita-icon-theme
```

Comment:
gnome applications depending directly on `adwaita-icon-theme` is something we're avoiding because there are so many alternative icon themes, so they should be put in `systemPackages`
```suggestion
```

Reply:
Upstream has a specific badge about "do not theme"; I'm not sure if that affects this.